Add how to report bugs section

christiam opened the pull request.

christiam committed:
Add how to report bugs section

tom6931 reviewed (commented):
In the "Table of Contents" there is a "How to report bugs?" link.  I see two problems.  First, it is indented under Maintainer but I'd expect it to be on the same level as Maintainer.  Second, the link just takes you to the top of the page.  Is there a typo in the URL?

I'm also concerned that the link to the BLAST+ problems (https://support.nlm.nih.gov/knowledgebase/article/KA-05184/en-us) is not a good fit for BLAST+ docker.  As a user, I'd be confused if I looked at this since about half the question don't really apply.  It's probably another set of somewhat similar questions.

christiam committed:
Fix TOC, refine instructions for bug reporting

christiam committed:
Merge branch 'master' into how-to-report-bugs

christiam committed:
Simplify TOC

christiam committed:
Simplify TOC

christiam commented:
I fixed the TOC as best as I could (it seems to be hand-edited these days) and added specific steps to follow from the article on how to report BLAST+.

christiam committed:
Customize instructions for reporting bugs

christiam committed:
Merge branch 'master' into how-to-report-bugs
